01_install_requirements.sh

```diff
@@ -13,9 +13,20 @@ fi
 # Update to latest packages first
 sudo yum -y update
 
+if grep -q "Red Hat Enterprise Linux release 8" /etc/redhat-release 2>/dev/null ; then
+    RHEL8=y
+fi
+
 # Install EPEL required by some packages
 if [ ! -f /etc/yum.repos.d/epel.repo ] ; then
-    if grep -q "Red Hat Enterprise Linux" /etc/redhat-release ; then
+    if [ "${RHEL8}" = "y" ] ; then
+        # TODO(russellb) Fix this when EPEL 8 is available
+        # sudo dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
+        #
+        # It's also possible we never need EPEL and everything we would have pulled from there is
```

Comment:
We better not use EPEL in our final product... We probably need it because of `pip` and we shouldn't really be using `pip` ;)

Reply:
Of course not.

I think it's only added in the centos 7 case because we pull some stuff from RDO, and RDO assumes EPEL is present, as well for some things.